docs/faq.md

```diff
@@ -266,3 +266,21 @@ Alternatively, perform a runtime check for the EarlGrey class:
 // Swift
 public static let envEarlGrey:Bool = NSClassFromString("EarlGreyImpl") != nil
 ```
+
+**How do I wait for an element to appear using Swift?**
+
+The best way is to [setup synchronization](features.md#synchronization) so that EarlGrey automatically waits
+for elements to appear. As a work around for when that's not possible, `GreyCondition` and `waitWithTimeout`
+are available. The following is an example of waiting for a collection view to populate.
+
+```swift
+GREYCondition(name: "Wait for collection view to populate", block: { _ in
+    var errorOrNil: NSError?
+
+    EarlGrey().selectElementWithMatcher(collectionViewMatcher)
+              .assertWithMatcher(grey_notNil(), error: &errorOrNil)
+
+    let success = errorOrNil == nil
+    return success
+}).waitWithTimeout(20.0)
```

Comment:
waitWithTimeout returns a success or failure in form of YES/NO. That return value should checked.